tty/uart16550/uart16550.c

```diff
@@ -204,8 +190,9 @@ static void uart_ioctl(unsigned port, msg_t *msg)
 }
 
 
-int _uart_init(unsigned int uartn, unsigned int speed, uart_t **uart)
+static int _uart_init(unsigned int uartn, unsigned int speed, uart_t **uart)
 {
+	unsigned int divisor = speed / 115200;
```

Comment:
Is this right? That would mean that we can not obtain baudrate lower than 115200 bps

Reply:
By looking at the old code - it should rather be `115200 / speed`.

Reply:
You're absolutely right. It's fixed now. Thanks for spotting this bug.